core/common/src/main/java/alluxio/grpc/GrpcChannel.java

```diff
@@ -127,7 +127,7 @@ public boolean isHealthy() {
 
   @Override
   public String toString() {
-    return MoreObjects.toStringHelper(this)
+    return MoreObjects.toStringHelper("")
```

Comment:
why is this (and others) ""?

Reply:
When given an instance, the string generated will start with that class's name. (Something like "GrpcChannel { Host=....... }". Giving empty header makes the traces much cleaner.

Reply:
We want to remove the class name? I would have thought the class name is useful?

Reply:
Yeah, I think it's better for this particular one. For the rest, empty is better.